macos/package_files/3.9.3/preinstall.sh

```diff
@@ -17,6 +17,18 @@ else
     launchctl setenv WAZUH_PKG_UPGRADE true
 fi
 
+if [ WAZUH_PKG_UPGRADE ]; then
+    mkdir -p $DIR/config_files/
+    cp -r $DIR/etc/{ossec.conf,client.keys,local_internal_options.conf,shared} $DIR/config_files/
+fi
+
+if [ WAZUH_PKG_UPGRADE ]; then
+    ETC_PKG=`pkgutil --pkgs | grep -i wazuh-etc`
+    if [ -z "$ETC_PKG" ]; then
```

Comment:
You can execute simplify even more this. Something like this should do the trick:
```
if pkgutil --pkgs | grep -i wazuh-etc > /dev/null 2&>1 ; then
    do stuff
fi
```

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/835644bb4e4316e8a8b751f9dd9192ea305f6352

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/cddd25dc5888f672a1ecd2c826ba8023e6f6e787